packages/web3-core/src/web3_config.ts

```diff
@@ -160,6 +212,11 @@ export abstract class Web3Config
 		this._config.transactionConfirmationBlocks = val;
 	}
 
+	/**
+	 * The `transactionPollingInterval` is used over HTTP connections. This option defines the number of seconds between Web3 calls for a receipt which confirms that a transaction was mined by the network.
+	 *
+	 * Default is `1000` ms
+	 */
 	public get transactionPollingInterval() {
```

Comment:
As you describing the property itself, don't need to repeat its name. 

```suggestion
	/**
	 * Used over HTTP connections. This option defines the number of seconds between Web3 calls for a receipt which confirms that a transaction was mined by the network.
	 *
	 * Default is `1000` ms
	 */
	public get transactionPollingInterval() {
```